packages/picasso.js/src/core/chart-components/text/text.js

```diff
@@ -47,9 +47,12 @@ function generateTitle({ title, definitionSettings, dock, rect, measureText, sty
     dy: 0,
     anchor: getTextAnchor(dock, definitionSettings.anchor),
     baseline: 'alphabetical',
+    stroke: style.overlay.stroke || 'transparent',
+    strokeWidth: style.overlay.strokeWidth || 0,
+    fontWeight: style.overlay.fontWeight || 400 ,
   };
 
-  extend(struct, style.text);
+  extend(struct, style.text, style.overlay);
```

Comment:
why have both style.text and style.overlay?
it looks like they are just merged together, why not just have all of it in style.text to begin with.
or are I missing somewhere where they are used differently?

Reply:
Just wanted to keep it separate as its only used for overlay labels and not the normal text, where below I have used an existing mixin : $label-overlay

Reply:
The existing label-overlay mixin is for a totaly different use case and it may be a bad idea to reuse it for this.

my current opinion is to not have a separate overlay style

Reply:
ok then I shall remove the mixin and use the specific styling we need inside text directly.